### PR TITLE
Fix anchors for Zola 0.15

### PR DESCRIPTION
**Pull request by AngelOnFira**
I'm on the latest version of Zola, and I had to remove these links since Zola appears to now check that anchors lead to somewhere properly. Good feature though!

### CODE REVIEW

File: content/news/016/index.md [hunk 1]
```diff
@@ -30,7 +30,6 @@ Feel free to send PRs about your own projects!
 
 Table of contents:
 
-- [Rust GameDev Podcast](#rust-gamedev-podcast)
```
Comment (17cupsofcoffee) [inline content/news/016/index.md:33]: This heading does exist, but it should be `[Rust GameDev Podcast #3](#rust-gamedev-podcast-3)`.

File: content/news/017/index.md [hunk 2]
```diff
@@ -30,7 +30,6 @@ Feel free to send PRs about your own projects!
 Table of contents:
 
 - [Rust Gamedev Meetup](#rust-gamedev-meetup)
-- [Rust GameDev Podcast](#rust-gamedev-podcast)
```
Comment (17cupsofcoffee) [inline content/news/017/index.md:33]: This heading does exist, but it should be `[Rust GameDev Podcast #4](#rust-gamedev-podcast-4)`.

**Review by 17cupsofcoffee — COMMENTED** [review]
:+1: to this - there's a couple of those headers that could be fixed rather than removed, though :)

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Nice!

**Pull request merged**